### PR TITLE
CMake: Tooling config fixes

### PR DESCRIPTION
**Pull request by dnakamura**

### CODE REVIEW

File: cmake/modules/platform/toolcfg/xlc.cmake [hunk 1]
```diff
@@ -1,66 +1,68 @@
...
+	set(CMAKE_C_ARCHIVE_CREATE "<CMAKE_AR> -X64 cr <TARGET> <LINK_FLAGS> <OBJECTS>")
+	set(CMAKE_C_ARCHIVE_FINISH "<CMAKE_RANLIB> -X64 <TARGET>")
+	
+elseif(OMR_HOST_OS STREQUAL "linux")
+	list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
+		-qxflag=selinux
+	)
+endif()
+
+list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qnortti -qnoeh)
```
Comment (rwy0717) [inline cmake/modules/platform/toolcfg/xlc.cmake]: I'd rather see this at the top near `OMR_PLATFORM_COMPILE_OPTIONS`, before conditional options.

**Comment by rwy0717** [comment]
LGTM

**Comment by dnakamura** [comment]
The `-fnortti -fno-exceptions` changes require #1583. Will remove them and bring them back in as a new pr when required changes have merged

**Pull request merged**